tests/pii-stress.manual.test.ts

```
@@ -0,0 +1,142 @@
+    if (MODEL) __setPiiModelOverrideForTests(MODEL);
+  });
+
+  test(
+    "prompt → redacted output across cases",
+    async () => {
+      const latencies: number[] = [];
+      console.log(
+        `\n##### PII redaction stress — model: ${MODEL ?? "default (gemini-3.1-flash-lite)"} #####`,
+      );
+
+      for (const c of CASES) {
```

ESLint

[error] Delete `··`

(prettier/prettier)